js/app/filters/SubscriptionListFilter.js

```diff
@@ -29,10 +29,10 @@ app.filter('subscriptionListFilter', function(CalendarListItem) {
 		}
 
 		return calendarListItems.filter(function(item) {
-			if (!CalendarListItem.isCalendarListItem(item)) {
+			if (!CalendarListItem.isSubscriptionListItem(item)) {
```

Comment:
You have to check for both calendarListItem and subscriptionListItem.
I have to admit that the naming is not the best here.

The Subscription Model is just about WebCal calendars.
Subscriptions in this filter are both webcal calendars and read-only calendars (like birthday-calendar and read-only shared-calendars)

I actually don't see a real need to distinguish between `CalendarListItem.IsCalendarListItem` and `CalendarListItem.isSubscriptionListItem`. What do you use this for?